app.js

```diff
@@ -2,22 +2,25 @@
 
 // List 3 expressions that surprised you or gave unexpected results, why?
 
-// List 2 exprsssions you felt comfortable interpreting at a glance.
+    // Lines 26, 46, 47, and 48 were all interesting to me, never seen those before.
 
-// List 1 expression you are still 'shaky' on, why?
+// List 2 exprsssions you felt comfortable interpreting at a glance.
 
+    //lines 44 and 25 were easy right away
 
+// List 1 expression you are still 'shaky' on, why?
 
+    // 22 & 23 are just complicated to work out in my mind
```

Comment:
Which ones are lines 22 and 23? Line 22 is blank. If you want anything cleared up just give me the specific example and I'll explain it!